src/landing-page/landing-page.js

```diff
@@ -0,0 +1,293 @@
+							'Export a zip file ready to be imported into another WordPress environment.',
+							'create-block-theme'
+						) }
+					</p>
+					<Button
+						variant="link"
+						onClick={ () => setCreateModalType( 'blank' ) }
+					>
+						{ __(
+							'Create a new Blank Theme',
+							'create-block-theme'
+						) }
+					</Button>
+					<p>
+						{ __(
+							'Start from scratch! Create a blank theme to get started with your own design ideas.',
+							'create-block-theme'
+						) }
+					</p>
+					<Button
+						variant="link"
+						onClick={ () => setCreateModalType( 'clone' ) }
+					>
+						{ sprintf(
+							// translators: %s: theme name.
+							__(
+								'Create a Clone of "%s"',
+								'create-block-theme'
+							),
+							themeStyleData?.name
+						) }
+					</Button>
+					<p>
+						{ __(
+							'Use the currently activated theme as a starting point.',
+							'create-block-theme'
+						) }
+					</p>
+					<Button
+						variant="link"
+						onClick={ () => setCreateModalType( 'child' ) }
+					>
+						{ sprintf(
+							// translators: %s: theme name.
+							__(
+								'Create a Child of "%s"',
+								'create-block-theme'
+							),
+							themeStyleData?.name
+						) }
+					</Button>
+					<p>
+						{ __(
+							'Make a theme that uses the currently activated theme as a parent.',
+							'create-block-theme'
+						) }
+					</p>
+				</VStack>
+				<VStack className="create-block-theme__landing-page__body__right-column">
+					<h3>{ __( 'About the Plugin', 'create-block-theme' ) }</h3>
+					<p>
+						{ __(
+							"Create Block Theme is a tool to help you make Block Themes using the WordPress Editor. It does this by adding tools to the Editor to help you create and manage your theme. Themes created with Create Block Theme don't require Create Block Theme to be installed on the site where the theme is used.",
+							'create-block-theme'
+						) }
+					</p>
+					<h3>
+						{ __( 'Do you need some help?', 'create-block-theme' ) }
+					</h3>
+					<p>
+						{ createInterpolateElement(
+							__(
+								'Have a question? Ask for some help in the <a>forums</a>.',
+								'create-block-theme'
+							),
+							{
+								a: (
+									<ExternalLink href="https://wordpress.org/support/plugin/create-block-theme/" />
+								),
+							}
+						) }
+					</p>
+					<p>
+						{ createInterpolateElement(
+							__(
+								'Found a bug? Report it on <a>GitHub</a>.',
+								'create-block-theme'
+							),
+							{
+								a: (
+									<ExternalLink href="https://github.com/WordPress/create-block-theme/issues/new" />
+								),
+							}
+						) }
+					</p>
+					<p>
+						{ createInterpolateElement(
+							__(
+								'Want to contribute? Check out the <a>project on GitHub</a>.',
+								'create-block-theme'
+							),
+							{
+								a: (
+									<ExternalLink href="https://github.com/WordPress/create-block-theme" />
+								),
+							}
+						) }
+					</p>
```

Comment:
It's interesting that it works by replacing the anchor with the component, but strictly we should use the same element:

```suggestion
						{ createInterpolateElement(
							__(
								'Want to contribute? Check out the <ExternalLink>project on GitHub</ExternalLink>.',
								'create-block-theme'
							),
							{
								ExternalLink: (
									<ExternalLink href="https://github.com/WordPress/create-block-theme" />
								),
							}
						) }
					</p>
```

Reply:
done